src/main/java/racingcar/domain/Cars.java

```diff
@@ -1,64 +1,72 @@
 package racingcar.domain;
 
 import java.util.ArrayList;
-import java.util.HashSet;
 import java.util.List;
-import java.util.Set;
-import racingcar.policy.RandomValueMovePolicy;
+import racingcar.utils.MovableCondition;
 
 public class Cars {
-    private static final int MIN_CAR_COUNT = 1;
-
     private final List<Car> values;
 
-    public Cars(List<String> names) {
-        validateCount(names.size());
-        validateUnique(names);
-
-        this.values = createCars(names);
+    public Cars(List<Car> inputs) {
+        validateNotEmpty(inputs);
+        this.values = new ArrayList<>(inputs);
     }
 
-    private void validateCount(int count) {
-        if (isInvalidCount(count)) {
-            throw new IllegalArgumentException("자동차 대수는 1이상이어야 합니다.");
+    private void validateNotEmpty(List<Car> inputs) {
+        if (inputs == null || inputs.isEmpty()) {
+            throw new IllegalArgumentException("자동차는 1대 이상이어야 합니다.");
         }
     }
 
-    private boolean isInvalidCount(int count) {
-        return count < MIN_CAR_COUNT;
-    }
-
-    private void validateUnique(List<String> names) {
-        Set<String> uniqueNames = new HashSet<>(names);
-
-        if (names.size() != uniqueNames.size()) {
-            throw new IllegalArgumentException("자동차 이름은 중복될 수 없습니다.");
-        }
+    public Cars(String inputNames) {
+        this(createCars(inputNames));
```

Comment:
```suggestion
    public Cars(String inputNames) {
        this(createCars(inputNames));
    }
    
    public Cars(List<Car> inputs) {
    }
```
주 생성자가 생성자의 마지막에 위치하는 것이 관례임